pkg/extensions/extension_search_test.go

```diff
@@ -53,16 +60,17 @@ func TestTrivyDBGenerator(t *testing.T) {
 		cveInfo := cveinfo.NewCVEInfo(storeController, repoDB, "ghcr.io/project-zot/trivy-db", "", logger)
 		generator := NewTrivyTaskGenerator(time.Minute, cveInfo, logger)
 
-		sch.SubmitGenerator(generator, 12000*time.Millisecond, scheduler.HighPriority)
+		sch.SubmitGenerator(generator, 1200*time.Millisecond, scheduler.HighPriority)
```

Comment:
This seems too often. Let's not download the DB every 1.2 seconds? It doesn't even have enough time to complete the download before the new one starts.